### ISSUE TITLE
Ask user if they want to separate home and root

### ISSUE THREAD

**Issue by matteoturini** [archlinux/archinstall#879]
Asking the user if they want to separate /home and / is a better idea than just making a 20Gb root partition. I install a lot of libraries so I will definetly pass 20Gb.

Why was this change made?

**Comment by Torxed** [comment]
It's was after users reporting that they wanted this that we introduced it. It had to do with more easily backup or replace the installation. However the defaults were obviously not sane for most users, and has since been tweaked to 60GB in v2.3.1 that's due to be released in a few days. And in `master` we've introduced a question for it (I think, I'm on my phone right now so can't verify).

I agree that this feature request have caused more issues than it solved issues, which is why we're partially reverting the introduction :)

Since we agree and have solved this for the upcoming version, I'll go ahead and close this.

**Comment by matteoturini** [comment]
Thanks for the answer.

**Comment by OJFord** [comment]
@Torxed Thanks, I installed a new system recently and was also surprised by this. 

I expanded deleted the home partition and expanded the root to fill the disk, but it seems the encrypted volume is set separately:

```
$ lsblk
NAME        MAJ:MIN RM   SIZE RO TYPE  MOUNTPOINTS
nvme0n1     259:0    0 465.8G  0 disk  
├─nvme0n1p1 259:1    0   484M  0 part  /boot
└─nvme0n1p2 259:2    0 465.3G  0 part  
  └─luksdev 254:0    0 465.3G  0 crypt /
```

```
$ df -h
Filesystem           Size  Used Avail Use% Mounted on
dev                   16G     0   16G   0% /dev
run                   16G  1.4M   16G   1% /run
/dev/mapper/luksdev   37G   28G  9.0G  76% /
tmpfs                 16G  452K   16G   1% /dev/shm
/dev/nvme0n1p1       484M  117M  367M  25% /boot
tmpfs                 16G  8.0K   16G   1% /tmp
tmpfs                3.2G   44K  3.2G   1% /run/user/1000
```

I found the wiki (uncharacteristically) unhelpful on this, I think since it makes quite a specific assumption on what you're trying to do (move to a larger disk).

Nevertheless I tried:
```
#   pvresize /dev/mapper/luksdev
  Failed to find physical volume "/dev/mapper/luksdev".
  0 physical volume(s) resized or updated / 0 physical volume(s) not resized
```

Any idea how I can expand the encrypted volume to fill the partition?

**Comment by OJFord** [comment]
> And in master we've introduced a question for it (I think, I'm on my phone right now so can't verify).

You have indeed, merged in #801, and in the 2.4.0 release candidates.

**Comment by OJFord** [comment]
> I deleted the home partition and expanded the root to fill the disk, but it seems the encrypted volume is set separately
> [...]
> Any idea how I can expand the encrypted volume to fill the partition?

(Bothered to) figure it out in the end, here's what I did in case it helps anyone else:
```shell
# cryptsetup resize luksdev
```

Reboot to installer medium:
```shell
# modprobe dm-crypt
# cryptsetup open /dev/nvme0n1p2 luksdev
# vgscan --mknodes
# vgchange -ay
# cryptsetup status luksdev  # note the size in sectors
# resize.f2fs -t <num sectors> /dev/mapper/luksdev
```

And reboot to installed OS.

The `cryptsetup resize` could presumably be done from installer (after `open`), and I'm not sure `vgscan`/`vgchange` was necessary, just reporting it as I did it.
